tests/system/test_dbapi.py

```diff
@@ -344,6 +344,50 @@ def test_DDL_autocommit(shared_instance, dbapi_database):
             op.result()
 
 
+def test_ddl_executemany(shared_instance, dbapi_database):
+    """Check that DDLs in autocommit mode are immediately executed."""
+
+    try:
+        conn = Connection(shared_instance, dbapi_database)
+        want_row = (
```

Comment:
Can we verify that the connection is in `autocommit=true` mode here? (The comment at the start of the test indicates that the connection should be in autocommit mode)

Reply:
Same as last comment about wrong comment